src/server.ts

```diff
@@ -18,9 +18,10 @@ export async function initServer() {
 
   const { url } = await startStandaloneServer(server, { listen: { port: +process.env.PORT! } });
   console.log(`🚀  Server ready at: ${url}`);
+  return server;
 }
 
 export async function run() {
   await initDatabase();
-  await initServer();
+  return await initServer();
```

Comment:
```suggestion
  return initServer();
```

Aqui não precisa do await pq o `initServer` já é uma Promise, e alguém vai dar `await` nesse `run` tbm, certo? Segue uma página com algumas dicas sobre o assunto: https://www.notion.so/taqtile/When-should-I-use-async-await-face75517dd64a1ba1da9e98393d8653?pvs=4